tests/integration_tests/test_missing_value_strategy.py

```diff
@@ -125,3 +125,76 @@ def test_missing_values_drop_rows(csv_filename, tmpdir):
     # run preprocessing
     ludwig_model = LudwigModel(config, backend=backend)
     ludwig_model.preprocess(dataset=df)
+
+
+@pytest.mark.parametrize(
+    "backend",
+    [
+        pytest.param("local", id="local"),
+        pytest.param("ray", id="ray", marks=pytest.mark.distributed),
+    ],
+)
+@pytest.mark.parametrize("outlier_threshold", [1.0, 3.0])
+@pytest.mark.parametrize("outlier_strategy", [None, "fill_with_mean", "fill_with_const"])
+def test_outlier_strategy(outlier_strategy, outlier_threshold, backend, tmpdir, ray_cluster_2cpu):
+    fill_value = 42
+    kwargs = {
+        PREPROCESSING: {
+            "outlier_strategy": outlier_strategy,
+            "outlier_threshold": outlier_threshold,
+            "fill_value": fill_value,
+        }
+    }
+    input_features = [
+        number_feature(**kwargs),
+    ]
+    output_features = [binary_feature()]
+
+    # Values that will be 1 and 3 std deviations from the mean, respectively
+    sigma1, sigma1_idx = -150, 4
+    sigma3, sigma3_idx = 300, 11
+
+    num_col = np.array([77, 24, 29, 29, sigma1, 71, 46, 95, 20, 52, 85, sigma3, 74, 10, 98, 53, 110, 94, 62, 13])
+    expected_fill_value = num_col.mean() if outlier_strategy == "fill_with_mean" else fill_value
+
+    input_col = input_features[0][COLUMN]
+    output_col = output_features[0][COLUMN]
+
+    bin_col = np.array([1, 0, 1, 1, 0, 1, 1, 0, 0, 1, 1, 1, 1, 0, 0, 1, 0, 0, 1, 0], dtype=np.bool_)
+    dataset_df = pd.DataFrame(
+        data={
+            input_col: num_col,
+            output_col: bin_col,
+        }
+    )
+
+    dataset_fp = os.path.join(tmpdir, "dataset.csv")
+    dataset_df.to_csv(dataset_fp)
+
+    config = {
+        "input_features": input_features,
+        "output_features": output_features,
+        TRAINER: {"epochs": 2, BATCH_SIZE: 128},
```

Comment:
nit: Don't really need this since we're just checking preprocessing

Reply:
True!